src/textual/screen.py

```diff
@@ -106,6 +106,19 @@ class Screen(Generic[ScreenResultType], Widget):
     Set to `""` to disable auto focus.
     """
 
+    CSS: ClassVar[str] = ""
+    """Inline CSS, useful for quick scripts. Rules here take priority over CSS_PATH.
+
+    Note:
+        This CSS is _not_ scoped to only apply to the screen.
```

Comment:
Can we rephrase to not use a negative? i.e. Say what it does apply to, not what it doesn't?